contentcuration/contentcuration/static/js/edit_channel/import/vuex/importModule.js

```diff
@@ -0,0 +1,25 @@
+var mutations = require('./importMutations');
+var actions = require('./importActions');
+var getters = require('./importGetters');
+
+module.exports = {
+  namespaced: true,
+  state: {
+    // An array of JSON-ified ContentNode Models, which is updated
+    // whenever a user selects a Node to import
+    itemsToImport: [],
+    // start | show_warning | finish | null
+    importStatus: null,
```

Comment:
So vuex 2 still doesn't have a means of encoding the schemas here? (e.g. with prop-like validation to enforce allowable values for `itemsToImport` and `importStatus`?)

These kinds of implicit contracts are one thing that always bothered me in vuex 1.